prac_08/button_event_demo.py

```diff
@@ -0,0 +1,18 @@
+from kivy.app import App
```

Comment:
Possibly add docstrings at the start of the file, otherwise this all looks pretty good